src/ResourceManager/Websites/Commands.Websites/Cmdlets/DeploymentSlots/SetAzureWebAppSlot.cs

```diff
@@ -145,26 +145,10 @@ public override void ExecuteCmdlet()
                         {
                             Location = location,
                             ServerFarmId = WebApp.ServerFarmId,
-                            Identity = parameters.Contains("AssignIdentity") && AssignIdentity ? new ManagedServiceIdentity("SystemAssigned", null, null) : WebApp.Identity,
+                            Identity = parameters.Contains("AssignIdentity") ? AssignIdentity ? new ManagedServiceIdentity("SystemAssigned", null, null) : new ManagedServiceIdentity("None", null, null) : WebApp.Identity,
                             HttpsOnly = parameters.Contains("HttpsOnly") ? HttpsOnly : WebApp.HttpsOnly
                         };
 
-                        Dictionary<string, string> appSettings = WebApp.SiteConfig?.AppSettings?.ToDictionary(x => x.Name, x => x.Value);
```

Comment:
@panchagnula why do we no longer need this section?

Reply:
@cormacpayne  we needed this earlier because the Assign Identity feature was using an AppSetting to configure disabling / enabling MSI - since we had no support for NONE. We added this line of code to make sure we don't overwrite all the current AppSettings while doing that - now we don't need to mess with AppSettings during Identity anymore so i removed it.